tools/appliance/definitions/systemvmtemplate/definition.rb

```diff
@@ -27,15 +27,15 @@
 architectures = {
     :i386 => {
         :os_type_id => 'Debian',
-        :iso_file => 'debian-7.9.0-i386-netinst.iso',
-        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.9.0/i386/iso-cd/debian-7.9.0-i386-netinst.iso',
-        :iso_md5 => 'e101a11ddb31f85acef542df1a49bf57',
+        :iso_file => 'debian-7.11.0-i386-netinst.iso',
+        :iso_src => 'http://cdimage.debian.org/cdimage/archive/7.11.0/i386/iso-cd/debian-7.11.0-i386-netinst.iso',
+        :iso_md5 => '75055a694508f5b891038ec12d703c9e',
```

Comment:
Would it be possible to extract these values to a properties/configuration file?  Separating data from code is not more hygenic, but it also reduces the risk of the change and makes it easier to inject new values for development/testing.